apps/expo-go/ios/Exponent/Kernel/DevSupport/EXPackagerLogHelper.swift

```diff
@@ -0,0 +1,145 @@
+// Copyright 2015-present 650 Industries. All rights reserved.
+
+import Foundation
+import React
+
```

Comment:
maybe rename this class a little bit. the name looks too generic for other purposes

Reply:
Do you have a suggestion? :)

Reply:
Ah, of course - I agree - I thought you meant the whole class :)